app/helpers/application_helper.rb

```diff
@@ -34,7 +34,10 @@ def generate_breadcrumbs(current_page, previous_pages)
   def page_header(heading = nil, &block)
     extra_content = block_given? ? capture(&block) : nil
 
-    render 'shared/page_header', heading: heading, extra_content: extra_content
+    render 'shared/page_header',
+           heading: heading,
+           show_switch_service: is_switch_service_banner_enabled?,
```

Comment:
do you need to pass `show_switch_service` in?
we use `is_switch_service_banner_enabled?` as global helper, don't we?

Reply:
This is rendering a partial - it shouldn't need to rely on global state - hence passing in the state